README.md

```diff
@@ -59,11 +59,9 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Budget App] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+**[budget-app]** is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
```

Comment:
Great job documenting your README using the new Microverse Template. 👏🏾 

- Kindly personalize the h3 heading `Microverse README Template` and make relevant to your project or otherwise remove it.
- The live demo link is broken, kindly make sure that your project is deployed successfully and update your README with a working link.
- Kindly write Future Features that you feel can be implemented in the future to improve your application.
- Please make sure that your README does not have any broken links. The link to your issues page is broken, add a working link to your issue page in your repository.
- The same thing applies to your link to your license. Please add a license to your file and link it in your README.
- Kindly make sure that the "Acknowledgements" section or similar gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).
- Since the FAQ section is OPTIONAL, if you do not have anything to add under the section kindly remove it to make your README more clean and professional.